resources/sync_gateway_configs/testfest_todo_cc.json

```diff
@@ -17,16 +17,16 @@
       "username":"data-bucket",
       "password": "password",
       "users": {
-        "traun": {"password": "pass", "admin_channels": ["traun"]},
-        "james": {"password": "pass", "admin_channels": ["james"]},
+        "jay": {"password": "pass", "admin_channels": ["jay"]},
+        "eunice": {"password": "pass", "admin_channels": ["eunice"]},
         "pasin": {"password": "pass", "admin_channels": ["pasin"]},
         "jens": {"password": "pass", "admin_channels": ["jens"]},
         "jim": {"password": "pass", "admin_channels": ["jim"]},
         "sridevi": {"password": "pass", "admin_channels": ["sridevi"]},
-        "raghu": {"password": "pass", "admin_channels": ["raghu"]},
-        "mod": {"password": "pass", "admin_roles": ["moderator"]},
+        "manasa": {"password": "pass", "admin_channels": ["manasa"]},
+        "sandy": {"password": "pass", "admin_roles": ["sandy"]},
```

Comment:
since SGW devs sometimes also participates the testfest, you may add their accounts as one shot instead of creating account during the testfest. just a suggestion.